### PR TITLE
Add subgraph paused status to index node API

### LINKED ISSUE #4667: [Feature] Add subgraph status (paused/active) to index-node API and graphman info
### Description

After #4642 , subgraphs can be paused and resumed using `graphman` it would be good to add the status (paused/active) to the index-node API and `graphman info`

### Are you aware of any blockers that must be resolved before implementing this feature? If so, which? Link to any relevant GitHub issues.

_No response_

### Some information to help us out

- [X] Tick this box if you plan on implementing this feature yourself.
- [X] I have searched the issue tracker to make sure this issue is not a duplicate.

### PR DESCRIPTION
**Pull request by incrypto32**
Closes #4667

### CODE REVIEW

File: node/src/manager/deployment.rs [hunk 1]
```diff
@@ -228,6 +229,10 @@ impl Deployment {
             if let Some(status) = status {
                 let chain = &status.chains[0];
                 rows.extend(vec![
+                    status
+                        .paused
+                        .map(|b| b.to_string())
+                        .unwrap_or("---".to_string()),
```
Comment (mangas) [inline node/src/manager/deployment.rs:235]: What is this?

**Review by mangas — APPROVED** [review]

File: node/src/manager/deployment.rs (hunk 1 above)
Comment (mangas) [reply node/src/manager/deployment.rs:235]: I still dont understand why --- rather than defaulting to a sensible value like "false"?

Comment (mangas) [reply node/src/manager/deployment.rs:235]: I'm approving to unblock but I'd rather change this, however I'll leave that decision to you

Comment (incrypto32) [reply node/src/manager/deployment.rs:235]: Its a placeholder for `None` the value is a `Option<bool>`

**Pull request merged**